EIPS/eip-1901.md

```diff
@@ -0,0 +1,75 @@
+
+## Motivation
+EVM-based blockchains lack a form RPC Specification. This proposal attempts to standardize such a specification in a way that is versionable, and both human and machine readable.
+
+Ethereum clients can expose RPC endpoints with different method signatures and cause compatability issues between clients.
+
+Developers need a reliable developer experience, and an industry standard way to describe Ethereum JSON-RPC 2.0 APIs.
+
+
+## Implementation
+### How do I Solve the problem?
+JSON-RPC APIs can support the OpenRPC specification by implementing a service discovery method that will return the [OpenRPC document](https://github.com/open-rpc/spec#openrpc-document) for the JSON-RPC API. The method MUST be named `rpc.discover`. The `rpc.` prefix is a reserved method prefix for [JSON-RPC 2.0 Specification](https://www.jsonrpc.org/specification) system extensions.
+
+### Use Case
+This is the vision for the use case of OpenRPC and how it would relate to multi-geth:
+
+![MultGethRpc-usecase](../assets/multi-geth-use-case.png)
+
+## Specification
+
+### What is OpenRPC?
+The [OpenRPC](https://github.com/open-rpc/spec) Specification defines a standard, programming language-agnostic interface description for [JSON-RPC 2.0](https://www.jsonrpc.org/specification) APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code, additional documentation, or inspection of network traffic. When properly defined via OpenRPC, a consumer can understand and interact with the remote service with a minimal amount of implementation logic, and share these logic patterns across use cases. Similar to what interface descriptions have done for lower-level programming, the OpenRPC Specification removes guesswork in calling a service.
+
+##### Structure
+This is the structure of an OpenRPC Document:
+
+![openrpc-spec-structure](../assets/OpenRPC_structure.png)
+
+
+## Rationale
+### Why would we do this?
+Services need to figure out how to talk to each other. If we really want to build the next generation of automation, then having up to date libraries, documented APIs, and modern tools are going to provide easy discovery, on-boarding, and enable end user and developer interaction.
+
+Use cases for machine-readable [JSON-RPC 2.0](https://www.jsonrpc.org/specification) API definition documents include, but are not limited to:
+
+- A common vocabulary and document will keep developers, testers, architects, and technical writers all in sync.
+- Server stubs/skeletons generated in many languages
+- Clients generated in many languages
+- Mock Server generated in many languages
+- Tests generated in many languages
+- Documentation Generation
+
+
+## Alternative
```

Comment:
I think this content could be folded into one of the prior sections (potentially the rationale).

Reply:
I still think this is the case, but it can be discussed later with more input from others.